src/App.js

```diff
@@ -1,26 +1,69 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, {Component} from 'react';
 import './App.css';
+import {Button, Input} from 'reactstrap';
+import Task from './components/Card/Card';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component{
+  state ={
+    taskArray: [{title: 'Finish the task'},{title: 'Get er done'}]
+  }
+
+  changeName = (event)  => {
+    console.log(event.target.value)
+    const newName = event.target.value
+    this.setState({title : newName})
+  }
+
+  createCard(event){
+    event.preventDefault();
+    const task = this.state.taskArray;
+    const title = this.state.title;
+    task.push({title})
+    console.log(title)
+    this.setState({taskArray : task})
+  }
+
+  removeTask = (event) => {
+    const list = this.state.taskArray
+    // const removed = list.filter(function(key) {
+    //   return list !== key
+    // });
+    
+    console.log(Object.keys(list))
+}
+
+  render() {
+      return(
+        <div className='container'>
+          <form>
+            <Input onChange={(event) => this.changeName(event)} name='Name of Input' 
+              placeholder='Card Title'
+              />
+            <Button type='submit'
+              color="primary" 
+              onClick={this.createCard.bind(this)}
+              >
+              Submit
+            </Button>
+          </form>
+          <div className='row'>
+          {this.state.taskArray.map((task, index) => {
```

Comment:
This makes it pretty hard to read. Can you put this outside the render and abstract it to a const